hash.c

```diff
@@ -4513,35 +4513,40 @@ any_p_i_pattern(VALUE key, VALUE value, VALUE arg)
 /*
  *  call-seq:
  *    hash.any? -> true or false
- *    hash.any?(object) -> true or false
+ *    hash.any?(entry) -> true or false
  *    hash.any? {|key, value| ... } -> true or false
  *
  *  Returns +true+ if any element satisfies a given criterion;
  *  +false+ otherwise.
  *
- *  If +self+ has no element, returns +false+ and argument or block
- *  are not used.
+ *  If +self+ has no element, returns +false+ and argument or block are not used;
+ *  otherwise behaves as below.
  *
  *  With no argument and no block,
- *  returns +true+ if +self+ is non-empty; +false+ if empty.
+ *  returns +true+ if +self+ is non-empty, +false+ otherwise.
  *
- *  With argument +object+ and no block,
+ *  With argument +entry+ and no block,
  *  returns +true+ if for any key +key+
- *  <tt>h.assoc(key) == object</tt>:
+ *  <tt>h.assoc(key) == entry</tt>, +false+ otherwise:
```

Comment:
Do you mean `self` here? `h` is not defined.

```suggestion
 *  <tt>self.assoc(key) == entry</tt>, +false+ otherwise:
```